Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,59 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+
+        StartOver:
+            InteractionManager.PerformInteractions();
+
+            Console.WriteLine("Whould you like to start again?");
+            var answer = Console.ReadLine();
+            if (answer.ToLower() == "y") goto StartOver;
+        }
+    }
+
+
+
+    public static class InteractionManager
+    {
+        public static void PerformInteractions()
+        {
+            var person = new Person
+            {
+                Name = Prompter("What is your full name?"),
+                Age = int.Parse(Prompter("What is your age?")),
+                Weight = double.Parse(Prompter("What is your weight in kg?")),
```

Comment:
For calculations of minor importance you can und should use float. That is good enough. :)